docs/features/datastores/faq.md

```diff
@@ -1,40 +1,63 @@
 ---
-title: FAQ
+title: DatastoreFAQ
 sidebar_label: FAQ
 ---
 
-# FAQ
+# Datastore FAQ
 
 ## Data is private
 
-Any datastore you upload to Grid can only be accessed **by you**. Grid does not look at or manipulate your data in anyway.
+Any datastore you upload to Grid can only be accessed **by you**. Grid does not look at or
+manipulate your data in anyway.
 
 ## Data retention
 
-If you delete your account, data will be deleted immediately.
+If you delete your account, all datastores you have created will be deleted immediately
+and permanently.
 
 ## Public vs private data
 
-When you create a datastore, you certify that you have the rights to that data. Only you have access to your datastore. It's up to you to make sure you have the rights to access that data.
+When you create a datastore, you certify that you have the rights to that data. Only you
+have access to your datastore. It's up to you to make sure you have the rights to access
+that data. 
 
-## Does Grid charge for Data Storage?
+**_PRODUCT TEAM: WE NEED A LINK TO OUR TERMS OF SERVICE HERE_**
 
-Grid does not charge for storing data. However, there is a limit of 25 datastores.
+## Does Grid charge for Data Storage? {#datastore-charging-and-limits}
 
-## Does Grid have a limit on Datastore size?
+Grid does not charge for storing data. However, there is a limit of 50 datastores
+(including any/all version of the same datastore name). If you attempt to exceed this
+limit, you will have to delete other datastores / versions before your `create` operation
+will succeed. 
 
-Yes Grid DataStore max size is 1.5 TB. For security reasons, we only allow you to upload data for 12 hours continuously. If your data is too big to upload in a 12 hour period, it will stop uploading. Initiating `grid datastore create <args...>` again will resume the upload.
+Please reach out to use at [support@grid.ai](mailto:support@grid.ai) or on the [community
+Slack](https://gridai-community.slack.com) if this limit poses challenges for your
+workflow; we're happy to assist with alternatives or discuss what a more appropriate usage
+tier would look like for your usage.
+
+## Does Grid have a limit on Datastore size? {#datastore-size-limit}
+
+Yes, Grid DataStore max size is 1.5 TB. For security reasons, we only allow you to upload
```

Comment:
No idea where it came from but what is the correct answer?

Reply:
I don't actually know. We've tested about 1TB before and I know it works, but there is technically no "hard limit". not sure how to word this

Reply:
@essiequoi please reword the way product sees fit. I think something like "Grid Datastores theoretically have no limit in size. However, they have only been performance tested up to ~1TB <insert testing results>". We should probably also have a mechanism for clearing old datastores and document that too @rlizzo + @essiequoi